components/create_comment/create_comment.jsx

```diff
@@ -792,9 +792,13 @@ export default class CreateComment extends React.PureComponent {
                 >
                     <EmojiPickerOverlay
                         show={this.state.showEmojiPicker}
-                        container={this.props.getSidebarBody}
+
+                        // - removing allows picker to be added on top
```

Comment:
do we need to keep this comment here?

Reply:
Most of my comments were just for a bit of context for you guys. It's not necessary to keep them. I can clean up if/when you want!

Reply:
I'm incline to clean it up.  Instead of commenting here, you may add it to your PR summary.

Reply:
Removed